1224-user_administration/xxxx-user_is_assigned_creator_role_according_to_settings_for_their_customer_institution.feature

```diff
@@ -0,0 +1,10 @@
+    @xxxx
+    Scenario Outline: User is assigned the Creator role according to settings for their Customer Institution
+        Given that the User is logged in
+        And the user's Customer Institution is set to assign the Creator role for users with Feide roles <FeideRole>
+        And the user's Feide account has roles <FeideRole>
+        Then the user is assigned the Creator role
```

Comment:
When they navigate to their Profile page(?)
Then they see they are assigned Creator role (can they?)

Reply:
ja, det er faktisk sant! good spot